### DISCUSSION TITLE
Embedded Systems

### DISCUSSION THREAD

**Discussion by FrazzledTurtle** [dailydotdev/daily#1529] [Feature Requests]
Hi,
I would like to recommend adding a new category for embedded systems development and/or for hardware engineering (PCB Design), even if people into these fields are more rare compared to the other ones. 
I also remember seeing a member of your team posting something about integrating a job posting/hunting section on DailyDev. I think that would be a great addition. People could have a "job space" to showcase their company's work and what they are looking for. Not necessarily looking to hire someone, but asking for help in their open-source projects, startups or non-profit organizations. Making this towards small projects and small businesses could differentiate DailyDev from becoming "corporate" like Linkedin and develop its identity to be more about developing your skills through contributions. Contributions could be translated to points or another category on it's own and be shown at every person's cards.

**Comment by idoshamun** [comment]
Hi @FrazzledTurtle,

Thanks for the suggestion. Please check this discussion https://github.com/dailydotdev/daily/discussions/1152
